change/@azure-msal-browser-1de1d356-3446-4ed8-bd1d-439763a7abaa.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "none",
```

Looks like there's no longer any changes to msal-browser